include/etc/confluent/docker/configure

```diff
@@ -14,10 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o nounset \
-    -o errexit \
-    -o verbose \
-    -o xtrace
+. /etc/confluent/docker/bash-config
```

Comment:
Is this file being made available from base?

Reply:
yep